_redirects

```diff
@@ -40,7 +40,7 @@ https://cert-manager.io/docs/usage/kubectl-plugin/#certificatesigningrequest htt
 # These rules are in place to capture traffic that is specifically referencing these source urls
 https://docs.cert-manager.io/en/latest/getting-started/index.html https://cert-manager.io/docs/tutorials/
 https://docs.cert-manager.io/en/latest/tutorials/acme/index.html https://cert-manager.io/docs/tutorials/
-https://docs.cert-manager.io/en/latest/tasks/issuers/index.html https://cert-manager.io/docs/configuration/
+https://docs.cert-manager.io/en/latest/tasks/issuers/* https://cert-manager.io/docs/configuration/
```

Comment:
you'll need to add the `:splat` if you wish for the path to be added to the redirected URL - ie

```
https://docs.cert-manager.io/en/latest/tasks/issuers/* https://cert-manager.io/docs/configuration/:splat
```

Reply:
That wasn't my intention, no. There's no obvious mapping of the names of individual html pages on the `docs.cert-manager.io` site to pages under `https://cert-manager.io/docs/` - I think using `:splat` would be the incorrect behaviour here?